polaris-react/src/components/Filters/Filters.scss

```
@@ -1,295 +1,261 @@
-  padding-top: var(--p-space-1);
-  // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
-  display: flex;
-
-  .open & {
-    // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
-    display: none;
+  &::after {
+    border-radius: $border-radius;
   }
-}
 
-.FilterTriggerLabelContainer {
-  // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
-  display: flex;
-  // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
-  align-items: center;
-  // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
-  justify-content: space-between;
-}
+  span {
+    margin-right: var(--p-space-05);
 
-.open {
-  &::before,
-  &::after {
-    content: '';
-    // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
-    position: relative;
-    // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
-    left: var(--p-space-4);
-    // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
-    width: calc(100% - var(--p-space-8));
     // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
-    height: var(--p-space-025);
-    background-color: var(--p-surface-subdued);
-    // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
-    display: block;
+    @media #{$p-breakpoints-md-up} {
+      margin-right: var(--p-space-025);
+    }
   }
 
-  &::before {
+  svg {
     // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
-    top: 0;
-  }
+    width: var(--p-space-5);
 
-  &::after {
     // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
-    bottom: 0;
+    @media #{$p-breakpoints-md-up} {
+      // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
+      width: var(--p-space-4);
+    }
   }
 }
 
-.open.first {
-  &::after {
-    content: '';
-    // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
-    bottom: 0;
+// stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
+@media #{$p-breakpoints-md-down} {
+  .FiltersWrapperWithAddButton {
     // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
     position: relative;
-    // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
-    left: var(--p-space-4);
-    // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
-    width: calc(100% - var(--p-space-8));
-    // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
-    height: var(--p-space-025);
-    background-color: var(--p-surface-subdued);
-    // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
-    display: block;
-  }
 
-  &::before {
-    // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
-    display: none;
+    .FiltersInner {
+      padding-top: calc(var(--p-space-2) + var(--p-space-025));
```

```suggestion
      padding-top: 2.25rem;
```

Could this be moved down from 9px to 8px?